include/dynamic_flag.h

```diff
@@ -285,6 +285,19 @@ ssize_t dynamic_flag_list_fprintf_cb(void *ctx, const struct dynamic_flag_state
  * It is safe if useless to call this function multiple times.
  */
 void dynamic_flag_init_lib(void);
+
+/**
+ * @brief Updates the "minimal write" flag (false by default).
+ *
+ * When the "minimal mode" flag is true, the dynamic_flag library
+ * attempts to minimise writes to binary code, in order to avoid
+ * copy-on-write.  This flag is false by default: fewer code paths
+ * means fewer hidden bugs.
+ *
+ * It is safe to call this function multiple times and concurrently
```

Comment:
actually, let's drop the "concurrently" guarantee: I don't feel like pulling in stdatomic.h for that.

Reply:
Done.